### ISSUE TITLE
limit of money withdraw 

### ISSUE THREAD

**Issue by imhinii** [stackup-dev/open-source-campaign#197]
**Name of Proposer**
HINI

**Is your feature request related to a problem? Please describe.**
yes there are many people who wants to withdraw few dollars sometime but its not possible now.
 
**Describe the solution you'd like**
remove the limit of 10 dollar to 5 dollar so its beneficial for both.

**Describe alternatives you've considered**
you can remove the limit instead of making it 5 dollar.

**Additional context**
 i do not have the screenshot of this feature because its not there in website of stack up

**Comment by sandeep-codes** [comment]
Yes there should be a custom withdrawal option so that anyone can withdraw amount according to their choice.
